### PR TITLE
Fixed the Inline dashboard filters issue

### PR DESCRIPTION
**Pull request by 4manasa**

### CODE REVIEW

File: packages/react-sdk-components/src/components/template/ListView/ListView.tsx [hunk 1]
```diff
@@ -337,14 +337,14 @@ export default function ListView(props /* : ListViewProps */) {
     let field = getFieldFromFilter(filterExpression, isDateRange);
     selectParam = [];
     // Constructing the select parameters list (will be sent in dashboardFilterPayload)
-    columnList.forEach(col => {
+    columnList.current.forEach(col => {
```
Comment (niallriddell) [inline packages/react-sdk-components/src/components/template/ListView/ListView.tsx:340]: You should guard against columnList.current being undefined columnList.current?

File: packages/react-sdk-components/src/components/template/ListView/ListView.tsx [hunk 2]
```diff
@@ -601,6 +581,27 @@ export default function ListView(props /* : ListViewProps */) {
   useEffect(() => {
     if (listContext.meta) {
       fetchDataFromServer();
+      setTimeout(() => {
```
Comment (niallriddell) [inline packages/react-sdk-components/src/components/template/ListView/ListView.tsx:584]: This doesn't feel right to me. Why do we need to use setTimeout here?  You're going to cause memory leaks becaue you're not unsubscribing in the useEffect cleanup.

**Review by niallriddell — CHANGES_REQUESTED** [review]
I've left some comments below

File: packages/react-sdk-components/src/components/template/ListView/ListView.tsx (hunk 1 above)
Comment (4manasa) [reply packages/react-sdk-components/src/components/template/ListView/ListView.tsx]: Added the chaining operator to handle undefined scenario

**Pull request merged**